### PR TITLE
[Azure Stack] Update fabric storage admin specs for new generation with autorest-beta

### PR DESCRIPTION
**Pull request by zyxyoshine**
1. Update package version to 2019-05-01
2. Add Top/Skip parameters for List
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by zyxyoshine** [comment]
@bganapa , this is to make generated module support Top/Skip params to align with previous Admin PowerShell module and update the package version to the latest. Please help review.

**Comment by zyxyoshine** [comment]
@bganapa could you help review this PR? Thanks.

File: specification/azsadmin/resource-manager/fabric/readme.azsautogen.md [hunk 1]
```diff
@@ -40,18 +40,16 @@ input-file:
...
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2016-05-01/MacAddressPool.json
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2016-05-01/NetworkOperationResults.json
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2016-05-01/ScaleUnit.json
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2016-05-01/ScaleUnitNode.json
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2016-05-01/SlbMuxInstance.json
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2016-05-01/StorageOperationResults.json
-  - $(this-folder)/Microsoft.Fabric.Admin/preview/2018-10-01/Drive.json
   - $(this-folder)/Microsoft.Fabric.Admin/preview/2018-10-01/StorageSubSystem.json
-  - $(this-folder)/Microsoft.Fabric.Admin/preview/2018-10-01/Volume.json
+  - $(this-folder)/Microsoft.Fabric.Admin/preview/2019-05-01/Drive.json
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/fabric/readme.azsautogen.md]: Are these new api-versions? in that case, can you please make a PR to master branch as well ?

Comment (bganapa) [reply specification/azsadmin/resource-manager/fabric/readme.azsautogen.md]: looks not, merging

**Review by bganapa — APPROVED** [review]

**Pull request merged**